commands/volumes.go

```diff
@@ -147,7 +147,15 @@ func RunVolumeCreate(c *CmdConfig) error {
 	region, err := c.Doit.GetString(c.NS, doctl.ArgVolumeRegion)
 	if err != nil {
 		return err
+	}
 
+	snapshotID, err := c.Doit.GetString(c.NS, doctl.ArgVolumeSnapshot)
+	if err != nil {
+		return err
+	}
+
+	if region == "" && snapshotID == "" {
+		return doctl.NewMissingArgsErr(c.NS)
```

Comment:
```suggestion
		errorMsg := fmt.Sprintf("%s.%s || %s.%s", c.NS, doctl.ArgVolumeRegion, c.NS, doctl.ArgVolumeSnapshot)
		return doctl.NewMissingArgsErr(errorMsg)
```

Good point on the error messaging. If we want to maintain a similar format, we could do something like this. Then the message would be:
> Error: (volume.create.region || volume.create.snapshot) command is missing required arguments

Reply:
Ah that's a great suggestion, thanks!